### PR TITLE
EDSC-1630 EDSC-1612 New project page for E2E services prototype.

### PR DESCRIPTION
**Pull request by mightynimble**
@jsiarto you can make all the changes in ```app/views/projects/new.html.erb``` and commit.

### CODE REVIEW

**Comment by mreese84** [comment]
What am I reviewing here?

**Comment by mightynimble** [comment]
@mreese84 all the clicks that used to lead to the project panel now should be sent users to the new project page.

**Comment by mreese84** [comment]
@mightynimble, I'm trying to test a saved project URL or a copy/pasted project URL with no luck. Can you confirm that this branch will direct users with existing project URLs to the new full screen project view?

**Comment by mightynimble** [comment]
@mreese84 I removed it and all the tests. It should be brought back when I work on EDSC-1636 though.

**Comment by mreese84** [comment]
Gotcha. This looks solid.

File: app/assets/stylesheets/modules/projects.css.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  .project-info-sidebar {
+    color: #fff;
+    width: 300px;
+    padding: 0 20px;
+    box-shadow: 5px 0 0 rgba(0,0,0,0.1);
+    z-index: 1;
+  }
+
+  .project-collections {
+    background: #e3e3e3;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/projects.css.scss]: @jsiarto, I think we should use color variables for most (all?) of these colors. Let's leverage variables and `darken`/`lighten` as much as possible and where appropriate.

**Comment by mightynimble** [comment]
Pushed a travis change to build ```e2e-services``` branch and PRs to the branch.

**Comment by mreese84** [comment]
@mightynimble, when you get a chance can you take a look at the failures on this PR?

**Comment by jboukes** [comment]
@mreese84 - I looked at the test failures, and I think it might be okay to rerun the job.  These failures are not 'normal' flaky tests (haven't seen these ones fail before); however, I can't replicate locally even when using the same seed.  Locally this branch is coming up green for that test (three times in a row).

Given that locally it is green even when using the same seed, I tend to feel those failures are just hiccups.

**Pull request merged**